garrysmod/lua/vgui/dhtml.lua

```diff
@@ -53,8 +53,8 @@ function PANEL:QueueJavascript( js )
 	end
 
 	self.JS = self.JS or {}
-
-	table.insert( self.JS, js )
+	self.JS[self.JS + 1] = js

```

Comment:
This needs to be changed

Reply:
This is wrong, it must be `self.JS[#self.JS + 1] = js`